src/common/actions/snap-builds.js

```diff
@@ -17,169 +21,52 @@ export function fetchBuildsError(error) {
   };
 }
 
-export function fetchBuilds(repository) {
+function checkStatus(response) {
+  if (response.status >= 200 && response.status < 300) {
+    return response;
+  } else {
+    return response.json().then(json => {
```

Comment:
They are optional, but I guess it would be good to be consistent about it from code style point of view.

Because you have to use parens when there are no args or multiple args (`() => { ... }`) I would agree to have a guideline to use parens always (for consistency).